### DISCUSSION TITLE
The weight for virial

### DISCUSSION THREAD

**Discussion by JunboLu** [deepmodeling/deepmd-kit#590] [Q&A] [answered]
Hi,
  I have a question about weight for virial. I see the examples in deepmd-kit example directory. I find that both start_prev_v and limit_prev_v are zero. So if I have virial data, how could I set weight for virial?
  With my best regards,
  Junbo

**Comment by tuoping** [comment, marked as answer]
The same setting for force will do.

**Reply by AnguseZhang** [reply]
@tuoping @JunboLu  Not force. One common settings is the same as energy.

**Reply by tuoping** [reply]
That was a slip. I'm terribly Sorry. Orz.
